configuration/spring-config-server/src/main/resources/application.yml

```
@@ -26,11 +26,8 @@ spring:
-
-
-
-
-
 server:
-  port: 8888
\ No newline at end of file
+  port: 8888
+
+encrypt:
+  key: "MySecretKey12345"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)